queries/unix_filespace.kusto

```diff
@@ -8,28 +8,30 @@ let VMs = (MonitoringResources_CL
 let mountpoints = dynamic(["/", "/var", "/boot", "/tmp", "/home", "/opt", "/backup", "/boot/efi"]);
 let warning = int(80);
 let critical = int(90);
-Perf
-| where ObjectName contains "Logical Disk"
-| where CounterName == "% Used Space"
-| where InstanceName in (mountpoints)
-| where CounterValue >= (warning)
+InsightsMetrics
+| extend Mountpoint = tostring(parse_json(Tags)["vm.azm.ms/mountId"])
+| where Namespace == "LogicalDisk"
+| where Name == "FreeSpacePercentage"
+| where Mountpoint  in (mountpoints)
+| extend InverseVal = (100 - Val)
+| where InverseVal >= (warning)
 | where TimeGenerated > ago(5m)
 | extend affected_object = toupper(tostring(split(_ResourceId, "/") [-1]))
-| summarize arg_max(TimeGenerated, *) by affected_object, InstanceName
+| summarize arg_max(TimeGenerated, *) by affected_object, Mountpoint
 | extend _ResourceId = toupper(_ResourceId)
-| project _ResourceId, affected_object, CounterValue, InstanceName
+| project _ResourceId, affected_object, InverseVal, Mountpoint
 | join kind= leftouter VMs
 on $left._ResourceId == $right.id_s
 | where tags_managedby_s =~ "q.beyond" or isempty(tags_managedby_s) 
-| where CounterValue >= (warning)
-| extend state = case(CounterValue >= (critical), "Critical", CounterValue <= (critical), "Warning", "")
-| extend threshold = case(CounterValue >= (critical), critical, CounterValue < (critical), warning, warning)
+| where InverseVal >= (warning)
+| extend state = case(InverseVal >= (critical), "Critical", InverseVal <= (critical), "Warning", "")
+| extend threshold = case(InverseVal >= (critical), critical, InverseVal < (critical), warning, warning)
 | extend monitor_package = "AZ_UX_SC_ManagedOSUnix"
 | extend monitor_name = "AZ_UNIX_FILESPACE"
 | extend monitor_description = strcat("Disk threshold has been reached. More or equal than ", (threshold), "% Diskspace used. Check the Diskspace of the Mountpoint, please.")
 | extend script_name = "n/a"
 | extend script_version = "n/a"
-| extend value = round(CounterValue, 2)
+| extend value = round(InverseVal, 2)
 | extend additional_information = strcat("more or equal than ", (threshold), "% Diskspace used. Managed by ",(tags_managedby_s),".")
-| extend affected_entity = InstanceName
-| project TimeGenerated =now(), _ResourceId, state, affected_object, monitor_package, monitor_name, monitor_description, script_name, script_version, threshold, value, affected_entity, additional_information
+| extend affected_entity = InverseVal
```

Comment:
The affected_entity was previously the Instance Name. Now it is the Invers Value. Should probably be Mountpoint